cassandra-sink/src/test/resources/cassandra.properties

```diff
@@ -0,0 +1,4 @@
+info.shortDescription = Ingests data into a Cassandra table.
+options_class = org.springframework.xd.cassandra.CassandraSinkOptionsMetadata
+base_packages = org.springframework.xd.cassandra
```

Comment:
What is the purpose of this `cassandra.properties` file, especially in the `test` scope ?